packages/esm-ward-app/src/types/index.ts

```diff
@@ -9,20 +9,17 @@ import type {
 } from '@openmrs/esm-framework';
 import type React from 'react';
 
-export interface WardPatientCardProps {
-  patient: Patient;
-  visit: Visit;
-  bed?: Bed;
-}
-
-export type WardPatientCardRow = React.FC<WardPatientCardProps>;
-export type WardPatientCardElement = React.FC<WardPatientCardProps>;
+export type WardPatientCardRow = React.FC<WardPatient>;
+export type WardPatientCardElement = React.FC<WardPatient>;
 
 // WardPatient is a patient admitted to a ward, and/or in a bed on a ward
 export type WardPatient = {
   patient: Patient;
-  visit?: Visit;
+  visit: Visit;
+  bed?: Bed;
   admitted: boolean;
```

Comment:
Tangential, but we could potentially remove admitted now and calculate that based on encounterAssigningToCurrentInpatientLocation being null or not.